forecast_solar/models.py

```diff
@@ -38,13 +47,23 @@ class Estimate:
     wh_days: dict[datetime, int]
     wh_hours: dict[datetime, int]
     watts: dict[datetime, int]
+    api_rate_limit: int
     api_timezone: str
 
     @property
     def timezone(self) -> str:
         """Return API timezone information."""
         return self.api_timezone
 
+    @property
+    def account_type(self) -> AccountType:
+        """Return API account_type information."""
+        if self.api_rate_limit == 60:
+            return AccountType.PERSONAL
+        elif self.api_rate_limit == 5:
```

Comment:
```suggestion
        if self.api_rate_limit == 5:
```

`elif` is not needed here, it matches the first and returns, or it gets here anyway